monitoring/collectors/jenkins_collector_adapter.py

```diff
@@ -0,0 +1,59 @@
+import requests
+from collector import ICollector
+from stats import BuildStats, StageStats
+
+
+class InvalidResponseCodeError(Exception):
+    pass
+
+
+class JenkinsCollectorAdapter(ICollector):
+
+    def __init__(self, job_name, build_url):
+        self.job_name = job_name
+        self.url = build_url
+
+
+    def collect(self):
+        resp = self.get_raw_stats_from_jenkins()
+        return self.convert_raw_stats_to_build_stats(resp)
+
+
+    def get_raw_stats_from_jenkins(self):
```

Comment:
See our comments for this class tests, but get_raw_stats_from_jenkins and convert_raw_stats_to_build_stats should probably be private and not tested directly.